expected/15.4/extra/aggregates.out

```diff
@@ -1782,9 +1842,130 @@ select t1.f1 from t1 left join t2 using (f1) group by f1;
+ f2 | count 
+----+-------
+(0 rows)
+
 --Testcase 432:
 drop foreign table t1, t2;
 --
+-- Test planner's selection of pathkeys for ORDER BY aggregates
+--
+-- Ensure we order by four.  This suits the most aggregate functions.
+explain (costs off)
+select sum(two order by two),max(four order by four), min(four order by four)
+from tenk1;
+         QUERY PLAN          
+-----------------------------
+ Aggregate
+   ->  Foreign Scan on tenk1
+(2 rows)
+
+-- Ensure we order by two.  It's a tie between ordering by two and four but
+-- we tiebreak on the aggregate's position.
+explain (costs off)
+select
+  sum(two order by two), max(four order by four),
+  min(four order by four), max(two order by two)
+from tenk1;
+         QUERY PLAN          
+-----------------------------
+ Aggregate
+   ->  Foreign Scan on tenk1
+(2 rows)
+
+-- Similar to above, but tiebreak on ordering by four
+explain (costs off)
+select
+  max(four order by four), sum(two order by two),
+  min(four order by four), max(two order by two)
+from tenk1;
+         QUERY PLAN          
+-----------------------------
+ Aggregate
+   ->  Foreign Scan on tenk1
+(2 rows)
+
+-- Ensure this one orders by ten since there are 3 aggregates that require ten
+-- vs two that suit two and four.
+explain (costs off)
+select
+  max(four order by four), sum(two order by two),
+  min(four order by four), max(two order by two),
+  sum(ten order by ten), min(ten order by ten), max(ten order by ten)
+from tenk1;
+         QUERY PLAN          
+-----------------------------
+ Aggregate
+   ->  Foreign Scan on tenk1
+(2 rows)
+
+-- Try a case involving a GROUP BY clause where the GROUP BY column is also
+-- part of an aggregate's ORDER BY clause.  We want a sort order that works
+-- for the GROUP BY along with the first and the last aggregate.
+explain (costs off)
+select
+  sum(unique1 order by ten, two), sum(unique1 order by four),
+  sum(unique1 order by two, four)
+from tenk1
+group by ten;
+         QUERY PLAN          
+-----------------------------
+ GroupAggregate
+   Group Key: ten
+   ->  Foreign Scan on tenk1
+(3 rows)
+
+-- Ensure that we never choose to provide presorted input to an Aggref with
+-- a volatile function in the ORDER BY / DISTINCT clause.  We want to ensure
+-- these sorts are performed individually rather than at the query level.
+explain (costs off)
+select
+  sum(unique1 order by two), sum(unique1 order by four),
+  sum(unique1 order by four, two), sum(unique1 order by two, random()),
+  sum(unique1 order by two, random(), random() + 1)
+from tenk1
+group by ten;
+         QUERY PLAN          
+-----------------------------
+ GroupAggregate
+   Group Key: ten
+   ->  Foreign Scan on tenk1
+(3 rows)
+
+-- Ensure consecutive NULLs are properly treated as distinct from each other
+select array_agg(distinct val)
+from (select null as val from generate_series(1, 2));
+ERROR:  subquery in FROM must have an alias
+LINE 2: from (select null as val from generate_series(1, 2));
```

Comment:
Is it expected correctly?

Reply:
changed to more constructive output in https://github.com/pgspider/sqlite_fdw/pull/97/commits/6ed4bf065bdbc2cd568fec0b235fe343310c852e , please verify @MinhLA1410 !

Reply:
Thank you for supporting. It's OK